### DISCUSSION TITLE
Wheel Support for Veikk A15 Pro?

### DISCUSSION THREAD

**Discussion by andreluizrcn** [OpenTabletDriver/OpenTabletDriver#2767] [General]
Hello! hope this message reaches you well!!, I would like to know if there's a future wheel support for Veikk A15 PRO model.
Thanks :)))

**Comment by jamesbt365** [comment]
This will eventually be done once otd gets wheel support, this is tracked in #1367.

**Reply by andreluizrcn** [reply]
That's great news!!!! hope it it all comes out soon!! :))
